### ISSUE TITLE
SigLIP 2: Multilingual Vision-Language Encoders with Improved Semantic Understanding, Localization, and Dense Features

### ISSUE THREAD

**Issue by WoojunePark** [WoojunePark/splat_papers#30]
## [SigLIP 2: Multilingual Vision-Language Encoders with Improved Semantic Understanding, Localization, and Dense Features](https://arxiv.org/abs/2502.14786)

**Status:** `to-read` | **Published:** 2025-02-20

### Abstract

> We introduce SigLIP 2, a family of new multilingual vision-language encoders that build on the success of the original SigLIP. In this second iteration, we extend the original image-text training objective with several prior, independently developed techniques into a unified recipe -- this includes captioning-based pretraining, self-supervised losses (self-distillation, masked prediction) and online data curation. With these changes, SigLIP 2 models outperform their SigLIP counterparts at all model scales in core capabilities, including zero-shot classification, image-text retrieval, and transfer performance when extracting visual representations for Vision-Language Models (VLMs). Furthermore, the new training recipe leads to significant improvements on localization and dense prediction tasks. We also train variants which support multiple resolutions and preserve the input&#39;s native aspect ratio. Finally, we train on a more diverse data-mixture that includes de-biasing techniques, leading to much better multilingual understanding and improved fairness. To allow users to trade off inference cost with performance, we release model checkpoints at four sizes: ViT-B (86M), L (303M), So400m (400M), and g (1B).

### Links

[arXiv](https://arxiv.org/abs/2502.14786) | [PDF](https://arxiv.org/pdf/2502.14786) | [HTML](https://arxiv.org/html/2502.14786) | [Code](https://github.com/google-research/big_vision/tree/main/big_vision/configs/proj/image_text/README_siglip2.md)

### Figures

![Figure](https://arxiv.org/html/2502.14786/x1.png)
*Figure 1: SigLIP 2 adds the captioning-based pretraining from LocCa [62] as well as self-distillation and masked prediction from SILC [45] and TIPS [38] (during the last 20% of training) to the sigmoid loss from SigLIP [71]. For some variants, the recipe additionally involves fine-tuning with data curation [61] or adaptation to native aspect ratio and variable sequence length [6, 12].*

![Figure](https://arxiv.org/html/2502.14786/x2.png)
*Figure 2: Per-language image-text retrieval performance for SigLIP, SigLIP 2 and mSigLIP on Crossmodal-3600 [58]. SigLIP 2 almost matches the performance of mSigLIP (SigLIP trained on multilingual data) despite performing substantially better on English vision-language tasks (Table 1).*

![Figure](https://arxiv.org/html/2502.14786/x3.png)
*Figure 3: Comparing the NaFlex (a single checkpoint per model size supporting native aspect ratio and variable sequence length/resolution) and the standard square-input SigLIP 2 variants which use a separate checkpoint for each sequence length/resolution. The sequence lengths annotated on the x-axis correspond to training sequence lengths for NaFlex. NaFlex interpolates fairly well between training resolutions, but does not extrapolate well (not shown).*

![Figure](https://arxiv.org/html/2502.14786/x4.png)
*Figure 4: Comparison of different vision encoders after training a Gemma 2 LLM for 50M steps with a frozen vision encoder (PaliGemma [7] stage 1), followed by fine-tuning the VLM on individual datasets (PaliGemma stage 3). SigLIP 2 performs better than SigLIP and AIMv2 [20] for different model sizes and resolutions. Same data as in Table 6.*

![Figure](https://arxiv.org/html/2502.14786/x5.png)
*Figure 5: 10-shot and 0-shot accuracy for geographically diverse object classification tasks (Dollar Street, GeoDE), as well as geolocalization (GeoDE country/region) and landmark localization (GLDv2) tasks. SigLIP 2 consistently performs better than SigLIP (see Table 8 for additional results).*

![Figure](https://arxiv.org/html/2502.14786/x6.png)
*Figure 6: Representation bias (association of random objects with gender; lower is better) for different models.*

---

### 📝 Reading Notes Template

Leave a comment using the sections below.
Bullet items under each heading are synced to the paper's YAML tags.
Any other text goes to `## My Notes`.
Heading level (`#` or `##`) and singular/plural spelling both work.

```
## inputs
- 

## outputs
- 

## methods
- 

Free-form reading notes here.
```

*Managed by splat-papers. Close this issue once you have read the paper.*
